src/TodoList.js

```diff
@@ -0,0 +1,63 @@
+import {useState} from 'react';
+import {TodoEdit} from './TodoEdit';
+import {TodosForm} from './TodosForm';
+import styled from 'styled-components';
+
+function TodoList() {
+  const [todos, setTodos] = useState([]);
+
+  const addTodo = todo => {
+    if (!todo.text || /^\s*$/.test(todo.text)) {
+      return;
+    }
+
+    const newTodos = [todo, ...todos];
+
+    setTodos(newTodos);
+    console.log(...todos);
+  };
+
+  const updateTodo = (todoId, newValue) => {
+    if (!newValue.text || /^\s*$/.test(newValue.text)) {
+      return;
+    }
+
+    setTodos(prev => prev.map(item => (item.id === todoId ? newValue : item)));
+  };
+
+  const removeTodo = id => {
+    const removedArr = [...todos].filter(todo => todo.id !== id);
+
+    setTodos(removedArr);
+  };
+
+  const completeTodo = id => {
+    let updatedTodos = todos.map(todo => {
+      if (todo.id === id) {
+        todo.isComplete = !todo.isComplete;
+      }
+      return todo;
+    });
+    setTodos(updatedTodos);
+  };
+
+  return (
+    <StyledTodoList>
```

Comment:
It's not a list, it's a `main` element - that's not a good naming then. What about "Wrapper" something?